cocotb/triggers.py

```diff
@@ -726,7 +726,7 @@ class First(_AggregateWaitable):
             t_ret = await First(t1, t2)
 
     .. note::
-        When using the old-style :keyword:`yield`-based coroutines, ``t = yield [a, b]`` was another spelling of
+        When using :ref:`generator-based coroutines <yield-syntax>`, ``t = yield [a, b]`` was another spelling of
```

Comment:
```suggestion
        In the old-style :ref:`generator-based coroutines <yield-syntax>`, ``t = yield [a, b]`` was another spelling of
```
Part tense doesn't read to well in the second half without some time marker